lib/s3routes/routes/routeGET.js

```diff
@@ -108,7 +108,7 @@ function routerGET(request, response, api, log, statsClient, dataRetrievalFn) {
                     }
                     log.end().addDefaultFields({ contentLength });
                     routesUtils.statsReport500(err, statsClient);
-                    return routesUtils.responseStreamData(err, request.headers,
+                    return routesUtils.responseStreamData(err, request.query,
                         resMetaHeaders, dataGetInfo, dataRetrievalFn, response,
```

Comment:
for both of those calls, just a empty object `{}` is being passed -- makes sense since I don't think you can set the response content headers in those api's.